protos/feast/core/FeatureService.proto

```diff
@@ -25,7 +30,13 @@ message FeatureServiceSpec {
 
   // List of features that this feature service encapsulates.
   // Stored as a list of references to other features views and the features from those views.
-  repeated FeatureViewProjection features = 3;
+  repeated string features = 6;
```

Comment:
so is there a convention here like `feature_table:feature`?

Reply:
Also, would it make more sense to use a feature reference instead of a string?

Reply:
yep that's the convention. I'll add a note on that expectation in the docstring. It should be a string instead of feature reference since we need to know which feature view the feature name is associated with.

Reply:
A feature reference contains a reference to its associated view/table https://github.com/feast-dev/feast/blob/master/protos/feast/serving/ServingService.proto#L52. I actually think the FeatureViewProjection object has a good structure for this use case, even if we remove it from the rest of the user facing API. Wouldn't it be safer and more convenient to split views/tables from feature names vs having to rely on string conventions?